Detectors/Passive/src/Hall.cxx

```diff
@@ -86,20 +87,41 @@ void Hall::createMaterials()
   // only media needed for geometry are created
 
   //  Stainless Steel
-  matmgr.Mixture("HALL", 50, "STAINLESS STEEL3", asteel, zsteel, 7.88, 4, wsteel);
-  matmgr.Medium("HALL", 50, "STST_C2", 50, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
+  matmgr.Mixture("HALL", kSTST_C2, "STAINLESS STEEL3", asteel, zsteel, 7.88, 4, wsteel);
+  matmgr.Medium("HALL", kSTST_C2, "STST_C2", 50, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
 
   //  Air
-  matmgr.Mixture("HALL", 55, "AIR2", aAir, zAir, dAir, 4, wAir);
-  matmgr.Medium("HALL", 55, "AIR_C2", 55, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
+  matmgr.Mixture("HALL", kAIR_C2, "AIR2", aAir, zAir, dAir, 4, wAir);
+  matmgr.Medium("HALL", kAIR_C2, "AIR_C2", 55, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
 
   // Concrete
-  matmgr.Mixture("HALL", 57, "CONCRETE2", aconc, zconc, 2.35, 10, wconc);
-  matmgr.Medium("HALL", 57, "CC_C2", 57, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
+  matmgr.Mixture("HALL", kCC_C2, "CONCRETE2", aconc, zconc, 2.35, 10, wconc);
+  matmgr.Medium("HALL", kCC_C2, "CC_C2", 57, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
 
   //  Iron
-  matmgr.Material("HALL", 52, "IRON", 55.85, 26., 7.87, 1.76, 17.1);
-  matmgr.Medium("HALL", 52, "FE_C2", 52, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
+  matmgr.Material("HALL", kFE_C2, "IRON", 55.85, 26., 7.87, 1.76, 17.1);
+  matmgr.Medium("HALL", kFE_C2, "FE_C2", 52, 0, isxfld, sxmgmx, tmaxfd, stemax, deemax, epsil, stmin);
+}
+
+void Hall::SetSpecialPhysicsCuts()
+{
+
+  // MaterialManager used to set physics cuts
+  auto& matmgr = o2::Base::MaterialManager::Instance();
+  // prepare available cuts
+  const o2::Base::MaterialManager::ECut cutgam = o2::Base::MaterialManager::kCUTGAM;
+  const o2::Base::MaterialManager::ECut cutele = o2::Base::MaterialManager::kCUTELE;
+  const o2::Base::MaterialManager::ECut cutneu = o2::Base::MaterialManager::kCUTNEU;
+  const o2::Base::MaterialManager::ECut cuthad = o2::Base::MaterialManager::kCUTHAD;
+
+  const Float_t cut1 = 1.e00;
+  const Float_t cut2 = 1.e-1;
+  const Float_t cut3 = 1.e-3;
```

Comment:
add a comment for these values, where do the values come from

Reply:
so far, they are ported from AliRoot. Need to figure out who was responsible there setting these cuts